lib/analytics-page/addon/application/template.hbs

```diff
@@ -1,14 +1,14 @@
 {{! using unsafeTitle here to avoid double encoding because the title helper does its own }}
 {{page-title (t 'analytics.pageTitle' nodeTitle=this.node.unsafeTitle)}}
-<div class='container' local-class='PageContainer'>
-    <div local-class='Counts' class='row'>
-        <div class='col-sm-4 panel panel-default' local-class='CountBox'>
+<div class='container' local-class='page-container'>
+    <div local-class='counts {{if this.isMobile 'mobile'}}'>
+        <div class='panel panel-default' local-class='count-box {{if this.isMobile 'mobile'}}'>
```

Comment:
While we're here, I wonder if we shouldn't get rid of these `panel`-related classes. It'll have to happen now or in another ticket, and this is small enough that it's probably worth just taking care of now.